src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -721,6 +727,8 @@ class PickPage extends Component {
       alertMessage,
     } = this.state;
     const { showOnly } = this.props;
+    const emptyPicksQuantity =
```

Comment:
`Count` or `Size` might be more suitable, because `Quantity` might mislead that this is some sort of item quantity.

Reply:
:+1: